cmd/src/server.rs

```diff
@@ -939,10 +939,21 @@ impl TiKVServer<RaftLogEngine> {
         let env = get_env(self.encryption_key_manager.clone(), None /*base_env*/).unwrap();
         let block_cache = self.config.storage.block_cache.build_shared_cache();
 
+        // Try to dump raft data.
+        let data =
+            crate::dump::check_and_dump_raft_db(&self.config.raft_engine.config().dir, env.clone());
```

Comment:
Why not use `config.raftstore.raftdb_path`?

Reply:
The whole raftdb could be large. Why not load entries into raft engine while scaning?

Reply:
> Why not use config.raftstore.raftdb_path?

Because when enable raft_engine, tikv ignore config.raftstore and don't set the path to a valid path.